test-scripting/game_rules.focs.py

```diff
@@ -5,7 +5,8 @@
     type=float,
     default=1.0,
     min=0.1,
-    max=10.0)
+    max=10.0,
```

Comment:
I'm surprised this (`,` with nothing after before the `)`) isn't a format error in Python...

Reply:
Trailing comma is ok for function calls and list literals.  For tuples with a single element, it is mandatory. 

```
(1) == 1  # True
(1, ) == (1, )  # True
```

I find this feature very useful, since it reduce diff when adding new argument/element.

Reply:
> it reduce diff when adding new argument/element.

You mean just when the params are on separate lines?

Reply:
On separate lines with coma:

```diff
- a
+ a,
+ b
```

```diff
+ b,
```